agent/bench-scripts/pbench-uperf

```diff
@@ -100,6 +100,7 @@ function usage {
 	printf -- "\t--client-node[s]=str[,str]              An ordered list of client NUMA nodes which should be used for CPU binding\n"
 	printf -- "\t\t\t\t   For both options above, the order must correspond with the --clients/--servers list\n"
 	printf -- "\t\t\t\t   To omit a specific client/server from binding, use a value of -1.\n"
+	printf -- "\t--uperf-server[s]=str[,str]             override remotehost value in the uperf xml file (default comes from --servers, required by vsock)\n"
```

Comment:
The `s` in `--uperf-servers` is not optional (see line 196).  (For the case of `--client-node[s]`, that line has _both_ options defined and both are referenced at line 349.)